src/Engine/src/Application.cpp

```diff
@@ -52,8 +52,10 @@ auto usa::Engine::Application::start(const std::string_view &title) -> void
 {
     sf::Event event{};
     sf::Uint32 style = sf::Style::Close;
+    sf::Uint32 fullscreen = sf::Style::Fullscreen;
+    sf::Uint32 resize = sf::Style::Resize;
 
-    style |= m_settings.fullscreen ? sf::Style::Fullscreen : sf::Style::Resize;
+    style |= m_settings.fullscreen ? fullscreen : resize;
```

Comment:
Do we really need 2 variables for this?

Reply:
That wasn't compiling on my computer cause of cast of different size